docs/pages/guides/environment-variables.mdx

```diff
@@ -12,7 +12,9 @@ Environment variables are key-value pairs configured outside your source code th
 
 ## Using the `extra` field
 
-In the app config, there is an `extra` property. This property is available when you publish your project with EAS Update and when you build with EAS Build.
+In the app config, there is an `extra` property. If your variables are non-sensitive and can be checked into source code, you can just add them to the app.json `extra` property for local builds.
```

Comment:
```suggestion
In the app config, there is an `extra` property. If your variables are non-sensitive and can be checked into the source code, you can add them to the `extra` property in **app.json**. This property is available when you build your project with EAS Build and publish the project with EAS Update.
```

The context of "where" the property is available is important to keep you here. Other changes suggested is as per our writing styleguide.

Reply:
Is this correct?  In my experience EAS builds need env vars in EAS.json, not app.json.